apps/schools/domains/common/components/containers/BaseLayoutComponents/BaseLayout/index.tsx

```diff
@@ -12,17 +12,68 @@ import { IBaseLayoutProps } from './interfaces'
 import { OrganizationSelect } from '../OrganizationSelect'
 import { useLayoutContext } from '@domains/user/providers/baseLayoutProvider'
 import { COLLAPSED_DIVIDER_WIDTH, COLLAPSED_LAYOUT_WIDTH, DIVIDER_WIDTH, LAYOUT_WIDTH } from './styles/styles'
+import { QueryStatuses } from '@domains/common/constants/Enums'
+import Image from 'next/image'
+import ExclamationCircleOutlined from '@public/icons/ExclamationCircleOutlined.svg'
+import { Button } from '@domains/common/components/button'
+import { handleChangeStatusInvitation } from '@domains/common/handlers/changeStatusInvitation'
+import { useChangeStatusMutation } from '@domains/query/redux/queryApi'
+import { useGetInvitationsQuery } from '@domains/employee/redux/employeeApi'
 
 const { Content, Sider } = Layout
 
 export const BaseLayout: React.FC<IBaseLayoutProps> = (props) => {
     const { isCollapsed, toggleCollapsed } = useLayoutContext()
+    const { data: invitations, refetch } = useGetInvitationsQuery({})
+    const [mutation] = useChangeStatusMutation()
 
     const { children } = props
 
+    invitations?.results.map((invite) => console.log(invite))
+
     return (
         <>
             <Layout className={styles.placement}>
+                <div className={styles.invitationsContainer}>
+                    {invitations?.results.map((invite) => {
+                        if (invite.status === QueryStatuses.SENT || invite.status === QueryStatuses.IN_PROGRESS) {
```

Comment:
it is better to send request with filter status==SENT, but there is no such logic on backend so it is ok